Assets/Expansion2/CityBanners/citybannerinstances.xml

```diff
@@ -15,8 +15,8 @@
                 <Grid ID="CityDetails" Texture="Banner_StatusFrame" Size="55,20" Anchor="C,T" AnchorSide="I,O" Offset="0,-6" SliceTextureSize="24,20" SliceCorner="9,2" SliceSize="5,15">
```

Comment:
Reminder: enable Hide Whitespace changes, otherwise it looks like all of expansion1 citybannerinstances was updated, and makes it look like more was done to citybannermanager.xml than actually was.